src/app/WriteClient.cpp

```diff
@@ -480,6 +480,7 @@ CHIP_ERROR WriteClient::OnMessageReceived(Messaging::ExchangeContext * apExchang
     }
     else
     {
+        StatusResponse::Send(Protocols::InteractionModel::Status::InvalidAction, apExchangeContext, false /*aExpectResponse*/);
```

Comment:
This is not handling the `HandleTimedStatus` case right, if that gets an unexpected message type...

Reply:
addressed, thanks